### ISSUE TITLE
wrong syntax for exec_pre in fio job file

### ISSUE THREAD

**Issue by johnsimcall** [gluster/gluster-colonizer#70]
Fio's **exec_pre** directive does not allow chaining commands with a simple ';'  Nor does it allow repeating the exec_pre directive (tested with fio-3.16)  Instead a shell/bash script would need to be created which calls the many commands requested.  Consider changing

https://github.com/gluster/gluster-colonizer/blob/master/ansible/g1-jobfile-fio.j2#L12
`exec_prerun=sync; echo 3 > /proc/sys/vm/drop_caches`

to

`exec_prerun=./some_script.sh`

with

```
$ cat ./some_script.sh
#!/bin/bash -x
sync
echo 3 > /proc/sys/vm/drop_caches
```

**Comment by dustinblack** [comment]
`exec_prerun` doesn't work at all with client/server tests, as I have recently discovered. See https://github.com/axboe/fio/issues/814

**Comment by dustinblack** [comment]
The above upstream fio issue turned out to be a permissions issue in my other test setup, so it looks like exec_prerun should work as expected.

In the course of testing that, I also discovered that you _can_ in fact chain commands together, but you have to use a sub-shell and `&&` command separators:

`exec_prerun=bash -c 'sync && echo 3 >/proc/sys/vm/drop_caches'`

**Comment by dustinblack** [comment]
Fixed with 30fabccc65d3331c5a1126837555911fce78aff8
